unified_range/utils.py

```diff
@@ -36,6 +36,10 @@ def transform_to_semver(unified_spec):
     :return: semver: string
     """
     operators = {"lt": "<", "lte": "<=", "gt": ">", "gte": ">="}
+
+    if any(op in unified_spec for op in operators.values()):
+        raise ValueError(
+            "semver range seems to be a npm semver range.")
```

Comment:
Change value error message to better explain what happened and do not include npm, the library should be agnostic to that - only semver as a concept https://semver.org/